python/core/qgsvectorlayer.sip

```diff
@@ -662,11 +662,11 @@ class QgsVectorLayer : QgsMapLayer
      * @param description
      * @param useAsDefault
      * @param uiFileContent
-     * @param msgError
+     * @param msgError (out)
      */
     virtual void saveStyleToDatabase( const QString& name, const QString& description,
                                       bool useAsDefault, const QString& uiFileContent,
-                                      QString &msgError );
+                                      QString &msgError /Out/ );
```

Comment:
Well I was serious. The API was non functional before (or at the very least seriously cumbersome). I doubt anyone used that method before.